firebase-web/src/main/java/io/spine/web/firebase/NodeValue.java

```diff
@@ -97,6 +106,18 @@ public ByteArrayContent toByteArray() {
         return result;
     }
 
+    /**
+     * Parses this node value as a message of the given type.
+     */
+    public <M extends Message> M as(Class<M> cls) {
+        if (originalJson != null) {
+            return originalJson.as(cls);
+        } else {
+            String jsonMessage = value.toString();
+            return fromJson(jsonMessage, cls);
```

Comment:
I would put this `fromJson` into `@see` section of the method's Javadoc — as more clarification is required on how the value is parsed etc.

It will be also helpful to explain how this `originalJson` (which I have previously requested to document along with the ctor) plays here.